apps/logging/request_logger.py

```diff
@@ -45,15 +110,28 @@ def extract_request_data(self, request):
             self.standard_log_data["auth_uuid"] = None
         self.standard_log_data.update(get_session_auth_flow_trace(request))
 
-    def format_for_output(self, data_dict):
-        merged_dict = {**self.standard_log_data, **data_dict}
-        return super().format_for_output(merged_dict)
+    def format_for_output(self, data_dict, cls=None):
+        # return str as is
+        if isinstance(data_dict, str):
```

Comment:
I feel like we shouldn't be allowing strings here. I think all log outputs should be in the JSON format. It makes it much easier for log parsers if it's always in a single format. It also makes searching in splunk easier. If we want to accept a string argument in our logger (which I'm still up in the air about) then it should be converted to a dict: 
```
if (isinstance(data_dict, str):
    data_dict = { 'message': data_dict }
```

Reply:
got impression that new logging api be applied to all loggings - including those plain string message logging, seems it is not the case.
will change those string message logging back to using python logging.